crates/db_views/src/post_view.rs

```diff
@@ -1835,6 +1835,7 @@ mod tests {
         actor_id: inserted_community.actor_id.clone(),
         local: true,
         title: "nada".to_owned(),
+        sidebar: None,
         description: None,
         updated: None,
         banner: None,
```

Comment:
These tests would be simpler if you assert only specific fields, instead of building a full dummy object. Then its not necessary to make a change every time some unrelated field changes.

Reply:
There's a lot of these, so we should probably do that later as part of a test cleanup.